python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -784,8 +788,8 @@ def __init__(self):
             for attr, item in self._schema.allItems():
                 ## If an object is hidden behind a getter method we can't assign a parent or defvalue so don't bother - rcurrie
                 if item.getProperties()['getter'] is None:
-                    defVal = self._schema.getDefaultValue(attr)
-                    self.setNodeAttribute(attr, defVal)
+                    setattr(self, attr, self._schema.getDefaultValue(attr))
```

Comment:
I would __MUCH__ rather see the setNodeAttribute fixed to be safe, or a the parent to be correctly assigned after the call to setattr. I know it's probably correctly set higher up in the copy process somewhere but I don't want to re-introduce places where we're not setting the parent as it was a pain to get things as _stable_ as they are with regards to this atm.
I'm rather concerned you say this breaks things. Does this simply break with a bad call to ```_setParent``` or does something else break in a new and exiting way here?